### PR TITLE
Feature/zen 16376

### PR DESCRIPTION
**Pull request by morr-zenoss**

### CODE REVIEW

File: central-query-lib/pom.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <artifactId>central-query-lib</artifactId>
+    <version>0.0.21</version>
+    <name>central-query-lib</name>
+    <url>http://maven.apache.org</url>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <zapp.version>0.0.16</zapp.version>
+        <metric-consumer.version>0.0.6</metric-consumer.version>
```
Comment (kindkid) [inline central-query-lib/pom.xml]: Does this need to be updated? https://github.com/zenoss/zenoss.metric.consumer/blob/develop/pom.xml#L8

File: central-query/pom.xml [hunk 2]
```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+        <zapp.version>0.0.16</zapp.version>
+        <metric-consumer.version>0.0.6</metric-consumer.version>
```
Comment (kindkid) [inline central-query/pom.xml]: Is this used?

Comment (morr-zenoss) [inline central-query/pom.xml]: Doesn't look like it. I'll remove it.

File: central-query-lib/pom.xml (hunk 1 above)
Comment (morr-zenoss) [inline central-query-lib/pom.xml]: Don't think so. I'll remove it.

File: central-query/pom.xml [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+        </repository>
+        <repository>
+            <id>zenoss-europa</id>
+            <name>Internal Zenoss Europa Repository</name>
+            <url>http://nexus.zendev.org:8081/nexus/content/repositories/releases</url>
+            <snapshots>
+                <updatePolicy>always</updatePolicy>
+            </snapshots>
+        </repository>
+    </repositories>
```
Comment (kindkid) [inline central-query/pom.xml]: Why is this is in the pom? I think our local (or devimg) boxes have this in the ~/.m2/settings.xml

Comment (morr-zenoss) [inline central-query/pom.xml]: Not sure about this - it was in the central-query .pom before.

File: central-query-lib/pom.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.1.4</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.jersey</groupId>
+            <artifactId>jersey-server</artifactId>
```
Comment (kindkid) [inline central-query-lib/pom.xml]: Why does central-query-lib need jersey-server? (as opposed to jersey-client)

Comment (morr-zenoss) [inline central-query-lib/pom.xml]: Seems to work fine with jersey-client. I'll update the .pom

**Pull request merged**